storage/innobase/buf/buf0buf.cc

```diff
@@ -742,6 +742,215 @@ bool buf_page_is_corrupted(bool check_lsn, const byte *read_buf,
 
 #ifndef UNIV_INNOCHECKSUM
 
+/** Memory Pressure */
+
+#ifdef __linux__
+mysql_pfs_key_t mem_pressure_thread_key;
```

Comment:
If `PLUGIN_PERFSCHEMA=NO`, `mysql_pfs_key_t` will not be defined here. I’d seriously consider using `std::mutex` here, because this is not expected to be a contended mutex, and it is not declared in `all_innodb_mutexes[]` either.